share/spice/amazon/button_products_item.handlebars

```diff
@@ -0,0 +1,31 @@
+<div class="tile  tile--pr tile--c has-detail  tile--{{parentId}} opt--t-xxs" data-link="{{url}}">
+    <div class="tile__media  tile__media--pr">
+        <img src="" data-src="{{{imageProxy img}}}" alt="{{title}}" class="tile__media__img  js-lazyload" />
+        {{#and showBadge meta.options.badge}}{{{include meta.options.badge}}}{{/and}}
+    </div>
+    <div class="tile__body  tile__body--pr">
+        {{#if url}}
+            {{{formatTitle heading el="h6" className="tile__title" classNameSec="tile__title--pr" href="url" optSub=true ellipsis=100}}}
+        {{else}}
+            {{{formatTitle heading el="h6" className="tile__title" classNameSec="tile__title--pr" optSub=true ellipsis=100}}}
+        {{/if}}
+    <div class="tile__tx  tile--pr__sub  one-line">
+        {{#and meta.options.price price}}
+            <span class="tile--pr__price  price">{{price}}</span>
+        {{/and}}
+        {{#and meta.options.price meta.options.brand price brand}}
+            <span class="sep  tile__sep"></span>
+        {{/and}}
+        {{#and meta.options.brand brand}}
+            <span class="tile--pr__brand">by {{brand}}</span>
+        {{/and}}
+    </div>
+    {{#if meta.options.rating}}
+    <div class="tile__tx  tile__rating  one-line">
+        {{{starsAndReviews rating reviewCount url_review true}}}
+    </div>
+    {{/if}}
+    </div>
+    <button class="tile__button btn">Shop Now</button>
```

Comment:
I guess the same point applies to the hard-coded English here, too? If this test is locked down to en_US then I guess it's ok?